docs/vsphere_ci/scripts/install-docker.ps1

```diff
@@ -1,3 +1,4 @@
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
```

Comment:
Before, the install-openssh.ps1 script was installing it, and it ran first.

Reply:
Yes but why are you adding it back? `NuGet` was needed before to install `sshutils` which is no longer required.

Reply:
`NuGet` provider is a requirement to install `docker`, as per [documentation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server); 

> If you're prompted to install the NuGet provider, type Y to install it as well.

Even using the  `-Force` flag with `Install-Module -Name DockerMsftProvider -Repository PSGallery`  a manual confirmation is needed. So to avoid that, `NuGet` provider  is getting installed first.

Reply:
Please call that out by linking to the doc as it is not immediately obvious why it is needed.